surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformStreamCorruptionIT.java

```diff
@@ -49,12 +49,42 @@ public void warningIsNotEmitted() throws VerificationException
     {
         OutputValidator validator = unpack( "/surefire-1614-stream-corruption" )
                 .executeTest()
-                .verifyErrorFree( 1 );
+                .verifyErrorFree( 2 );
```

Comment:
Did you change the project within `surefire-1614-stream-corruption`? I do not understand why `2` tests.

Reply:
You're right, this line should not have changed. I first tried to add a new test in `surefire-1614-stream-corruption` (that's where the `2` comes from) but in the end, I created a new mini-project in `surefire-1659-stream-corruption` because the easiest way to trigger the problem is to have a `junit-platform.properties` and a redirection from `java.util.logging` to any other logging framework (I tested with both log4j and slf4j/logback). I'll change that `2` back to `1` ASAP. Do you have any other remark?

Reply:
Thx for making it clear. Yes there are some little changes only.